panel/src/components/Views/Pages/PageChangesView.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+	<k-panel class="k-panel-inside k-page-changes-view">
+		<header class="k-page-changes-header">
+			<k-button-group>
+				<k-button
+					:link="link"
+					:responsive="true"
+					:text="$t('back')"
+					icon="angle-left"
+					size="sm"
+					variant="filled"
+				/>
+				<k-button
+					:icon="modes[mode].icon"
+					:dropdown="true"
+					:responsive="true"
+					:text="$t('view')"
+					size="sm"
+					variant="filled"
+					@click="$refs.layout.toggle()"
+				/>
+				<k-dropdown-content ref="layout" :options="dropdown" />
+			</k-button-group>
+
+			<k-form-controls
+				:editor="editor"
+				:is-locked="isLocked"
+				:is-unsaved="isUnsaved"
+				:modified="modified"
+				@discard="onDiscard"
+				@submit="onSubmit"
+			/>
+		</header>
+		<main class="k-page-changes-grid" :data-mode="mode">
+			<section v-if="mode === 'latest' || mode === 'compare'">
+				<k-headline>
+					{{ modes.latest.label }}
+					<k-button
+						:link="src.latest"
+						icon="open"
+						size="xs"
+						target="_blank"
+						variant="filled"
+					/>
+				</k-headline>
+				<iframe :src="src.latest"></iframe>
+			</section>
+			<section v-if="mode === 'changes' || mode === 'compare'">
+				<k-headline>
+					{{ modes.changes.label }}
+					<k-button
+						v-if="isUnsaved"
+						:link="src.changes"
+						icon="open"
+						size="xs"
+						target="_blank"
+						variant="filled"
+					/>
+				</k-headline>
+				<iframe v-if="isUnsaved" :src="src.changes"></iframe>
+				<k-empty v-else>{{ $t("lock.unsaved.empty") }}</k-empty>
+			</section>
+		</main>
+	</k-panel>
+</template>
+
+<script>
+import PageView from "@/components/Views/Pages/PageView.vue";
+
+export default {
+	extends: PageView,
+	props: {
+		src: Object
+	},
+	data() {
+		return {
+			mode: "compare"
+		};
+	},
+	computed: {
+		modes() {
+			return {
+				latest: {
+					label: this.$t("version.latest"),
+					icon: "layout-right",
+					current: this.mode === "latest",
+					click: () => (this.mode = "latest")
+				},
+				changes: {
+					label: this.$t("version.changes"),
+					icon: "layout-left",
+					current: this.mode === "changes",
+					click: () => (this.mode = "changes")
+				},
+				compare: {
+					label: this.$t("version.compare"),
+					icon: "layout-columns",
+					current: this.mode === "compare",
+					click: () => (this.mode = "compare")
+				}
```

Comment:
`Suggestion:` I'd flip the order, to have the compare mode on top as the default one and then the more specific ones further below.